services/virus_scan/clamav_mod.conf

```diff
@@ -131,6 +131,33 @@ clamav_mod.MaxScanSize 100M
 # Default:
 #       clamav_mod.PhishingAlwaysBlockCloak off
 
+# TAG: clamav_mod.ProtectCCNumSSN
+# Format: clamav_mod.ProtectCCNumSSN on|off
+# Description:
+#	Enable the clamav data loss prevention (DLP) module which scans
+#	for credit card and SSN numbers. i.e. alert when detecting
+#	personal information.
+#	WARNING: This will break many government, banking and ecommerce
+#	sites. Use with Caution.
+# Default:
+#       clamav_mod.ProtectCCNumSSN off
+
+# TAG: clamav_mod.ProtectSSNNormal
+# Format: clamav_mod.ProtectSSNNormal on|off
+# Description:
+#	Search for [and alert when detecting] SSNs formatted as xx-yy-zzzz.
+#	Implies ProtectCCNumSSN on.
+# Default:
+#       clamav_mod.ProtectSSNNormal off
+
+# TAG: clamav_mod.ProtectSSNStripped
+# Format: clamav_mod.ProtectSSNStripped on|off
+# Description:
+#	Search for [and alert when detecting] SSNs formatted as xxyyzzzz.
+#	Implies ProtectCCNumSSN on.
+# Default:
+#       clamav_mod.ProtectSSNStripped off
+
 # TAG: clamav_mod.ReportVirusOnFailure
```

Comment:
Thank you for credit card numbers settings.
However I did not include them in the final patch, it is not make sense for a web antivirus system.